src/features/user/ManageProjects/components/microComponents/ProjectLocationMap.tsx

```diff
@@ -0,0 +1,143 @@
+import type {
+  MapLayerMouseEvent,
+  ViewState,
+  ViewStateChangeEvent,
+} from 'react-map-gl-v7/maplibre';
+import type {
+  UseFormClearErrors,
+  UseFormSetError,
+  UseFormSetValue,
+} from 'react-hook-form';
+import type {
+  BaseFormData,
+  ProjectCoordinates,
+  TreeFormData,
+} from '../BasicDetails';
+import type { SetState } from '../../../../common/types/common';
+import type {
+  ExtendedMapLibreMap,
+  MapLibreRef,
+} from '../../../../common/types/map';
+
+import { useCallback, useEffect, useRef, useState } from 'react';
+import MapGL, { NavigationControl, Marker } from 'react-map-gl-v7/maplibre';
+import 'maplibre-gl/dist/maplibre-gl.css';
+import getMapStyle from '../../../../../utils/maps/getMapStyle';
+import { getAddressFromCoordinates } from '../../../../../utils/geocoder';
+import { ProjectLocationIcon } from '../../../../../../public/assets/images/icons/projectV2/ProjectLocationIcon';
+import themeProperties from '../../../../../theme/themeProperties';
+import { useTranslations } from 'next-intl';
+import { centerMapOnCoordinates } from '../../../../../utils/projectV2';
+import {
+  DEFAULT_MAP_STATE,
+  DEFAULT_VIEW_STATE,
+} from '../../../../../utils/mapsV2/mapDefaults';
+
+interface ProjectLocationMapProps {
+  clearErrors: UseFormClearErrors<BaseFormData | TreeFormData>;
+  setError: UseFormSetError<BaseFormData | TreeFormData>;
+  setValue: UseFormSetValue<BaseFormData | TreeFormData>;
```

Comment:
Please use `ConservationFormData` for clarity - even though it is an alias of `BaseFormData`